test/resources/case6_placement_propagation/case6-test-policy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: case6-test-policy
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policies.ibm.com/v1alpha1
```

Comment:
Is this old API intentional?

Reply:
It's a copy paste from the case 1 policy.  So it was intentional sort of.